scripts/setup-archlinux.sh

```diff
@@ -61,3 +64,4 @@ echo
 echo "- ncurses5-compat-libs"
 echo "- makedepend"
 echo "- python2"
+echo "- lib32-c-ares"
```

Comment:
Also regarding lib32-c-ares on AUR, this is currently being maintianed by me, incase anyone wants themselves to be added as comaintainer, I would be happy to do so.